docker/io-lollipop/Dockerfile

```
@@ -0,0 +1,21 @@
+FROM functions-node-20 AS base
+
+WORKDIR /usr/src/app
+COPY . /usr/src/app
+
+RUN yarn install --immutable
+RUN yarn build
+
+FROM base AS runner
```

```suggestion
FROM node:20.12.2-alpine AS runner
```
using multi-stage build can significantly reduce image size. during a test with `io-session-manager` I managed to reduce the image size from ~2GB to 540MB. please consider adding this optimization and test it after you have done other changes since it can cause the container to fail